roles/big-service/files/big-service.py

```diff
@@ -0,0 +1,55 @@
+#! /usr/bin/env python2.7
+
+import BaseHTTPServer
```

Comment:
Using BaseHTTPServer is an interesting approach, I feel that using a framework like flask (http://flask.pocoo.org/) might have save you some time (and code 😄 )